src/Altinn.Apps/AppTemplates/AspNet/App/logic/App.cs

```diff
@@ -129,7 +155,7 @@ public override Task<AppOptions> GetOptions(string id, AppOptions options)
         /// <returns></returns>
```

Comment:
Maybe remove the return comment rather than having it be empty ?

Reply:
Added some text.